pkg/await/await.go

```diff
@@ -328,6 +376,20 @@ func Deletion(
 	case coreV1ResourceQuota:
 	case coreV1Secret:
 	case coreV1Service:
+	case coreV1ServiceAccount:
+	case extensionsV1Beta1Ingress:
+	case rbacAuthorizationV1ClusterRole:
+	case rbacAuthorizationV1ClusterRoleBinding:
+	case rbacAuthorizationV1Role:
+	case rbacAuthorizationV1RoleBinding:
+	case rbacAuthorizationV1Alpha1ClusterRole:
+	case rbacAuthorizationV1Alpha1ClusterRoleBinding:
+	case rbacAuthorizationV1Alpha1Role:
+	case rbacAuthorizationV1Alpha1RoleBinding:
+	case rbacAuthorizationV1Beta1ClusterRole:
+	case rbacAuthorizationV1Beta1ClusterRoleBinding:
+	case rbacAuthorizationV1Beta1Role:
+	case rbacAuthorizationV1Beta1RoleBinding:
```

Comment:
A shame that we have to repeat this large block of constants three times here, and looks like this could be fragile to forgetting to add/remove in one of these places in the future causing a subtle bug.  Since this appears to be here just to be able to error in case of unknown kinds, can the logic be inverted, or the ID looked up early and compared against a single maintainer list of supported kinds so that this doesn't need to be inlined in each of the operations:?

Reply:
Sure, let's do it in a follow up PR.